### PR TITLE
Separate devrel resources & Mindee informations

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

**Comment by fharper** [comment]
The test will pass when we merge this PR: https://github.com/mindee/devrel/pull/273

File: resources/README.md [hunk 1]
```diff
@@ -67,13 +67,14 @@ Mindee is an API first platform that helps developers automate applications' wor
...
-- [Newsletter](https://cdn.forms-content.sg-form.com/13b540c7-2c28-11ec-a526-325185d2b4f9)
+- [Newsletter](https://newsletter.mindee.com)
 - [SDK Node.js](https://github.com/mindee/mindee-api-nodejs)
 - [SDK Python](https://github.com/mindee/mindee-api-python)
-- [Slack Community](https://join.slack.com/t/mindee-community/shared_invite/zt-uzgmljfl-MotFVfH~IdEZxjp~0zldww)
+- [Slack Community](https://slack.mindee.com)
 - [Twitter](https://twitter.com/mindeeAPI)
 - [Website](https://mindee.com)
 - [YouTube](https://www.youtube.com/channel/UCXcb0H4P81RqvvvFfWdszoA) (video tutorials)
+Add COmputer VVision
```
Comment (fakela) [inline resources/README.md:78]: There is minor typo here

**Review by fakela — CHANGES_REQUESTED** [review]

File: resources/README.md (hunk 1 above)
Comment (fharper) [reply resources/README.md:78]: It shouldn't be there, thanks

Comment (fharper) [reply resources/README.md:78]: Oh actually, not sure why this README is the old one!

**Comment by fharper** [comment]
That PR is a shitshow... and I was talking about attention to details this morning 😂🤦‍♂️

**Comment by fakela** [comment]
> That PR is a shitshow... and I was talking about attention to details this morning 😂🤦‍♂️

It happens to the best of us 😂

**Review by fakela — APPROVED** [review]

**Pull request merged**